R/exploratoryfactoranalysis.R

```diff
@@ -281,7 +289,13 @@ exploratoryFactorAnalysis <- function(jaspResults, dataset, options, ...) {
 
   if (!ready) return()
 
-  mar <- psych::mardia(dataset, plot = FALSE)
+  mar <- try(psych::mardia(dataset, plot = FALSE), silent = TRUE)
+  if (isTryError(mar)) {
+    errmsg <- gettextf("Mardia test failed. Internal error message: %s", mar[1])
+    # with .extractErrorMessage the error is not informative because it contains multiple colons
```

Comment:
fyi: this behaviour of `.extractErrorMessage` will be changed (see https://github.com/jasp-stats/jaspBase/pull/76), so you can probably use it here as well